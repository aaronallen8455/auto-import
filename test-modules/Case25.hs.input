module Case25 where

main :: IO ()
main = Both.a
