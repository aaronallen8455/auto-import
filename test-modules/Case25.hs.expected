module Case25 where

import qualified M1 as Both
import qualified M2 as Both

main :: IO ()
main = Both.a
